### PR TITLE
Latest terms

### PR DESCRIPTION
**Pull request by kidehen**
Updated definition of name and fixed a few relations.

### CODE REVIEW

File: GlossaryOfTerms.ttl [hunk 1]
```diff
@@ -38,7 +39,7 @@ sourceDAV:
     schema:description """Description of OpenLink Software Glossary of Terms."""@en ;
     schema:author <http://www.openlinksw.com/#this> ;
     schema:dateCreated "2014-01-03T13:00:00-05:00"^^xsd:dateTime ;
-    schema:dateModified "2021-10-14T16:45:00-05:00"^^xsd:dateTime ;
+    schema:dateModified "2021-10-12T10:35:00-05:00"^^xsd:dateTime ;
```
Comment (TallTed) [inline GlossaryOfTerms.ttl:42]: This update was applied in March 2022, but the `schema:dateModified` was set to October 2021?  This appears to be part of the effective revert of the [previous commit from me](https://github.com/OpenLinkSoftware/glossaries/commit/cac9b5a2767189b21c2e144ef01b492d796ad95b) (applied by @openlink).

File: GlossaryOfTerms.ttl [hunk 2]
```diff
@@ -50,7 +51,7 @@ source:
     schema:description """Covers the definition of a variety of terms associated with realm of
                       Data, Networks, and Cloud abstractions."""@en ;
     schema:dateCreated "2014-01-03T13:00:00-05:00"^^xsd:dateTime ;
-    schema:dateModified "2021-10-14T16:45:00-05:00"^^xsd:dateTime ;
+    schema:dateModified "2021-10-12T10:35:00-05:00"^^xsd:dateTime ;
```
Comment (TallTed) [inline GlossaryOfTerms.ttl:54]: This update was applied in March 2022, but the `schema:dateModified` was set to October 2021?  This appears to be part of the effective revert of the [previous commit from me](https://github.com/OpenLinkSoftware/glossaries/commit/cac9b5a2767189b21c2e144ef01b492d796ad95b) (applied by @openlink).

File: GlossaryOfTerms.ttl [hunk 3]
```diff
@@ -3864,11 +3879,10 @@ owl:IrreflexiveProperty
...
     skos:altLabel "SPARQL" ;
-    schema:description """Recursive acronym of "SPARQL Protocol and RDF Query Language", SPARQL comprises
-                          a Declarative Structured Query language for RDF-model-based structured data, Query
-                          Results Serialization Formats, HTTP-based Query Service Protocol, and an HTTP-based
-                          Database- (or Store-) oriented protocol for Create, Read, Update, and Delete (CRUD)
-                          operations."""@en ;
+    schema:description """Recursive acronym that covers: a Declarative Structured Query language for
+                      RDF-model-based structured data, Query Results Serialization Formats,
+                      HTTP-based Query Service Protocol, and an HTTP-based Database- (or Store-)
+                      oriented protocol for Create, Update, and Delete (CRUD) operations."""@en ;
```
Comment (TallTed) [inline GlossaryOfTerms.ttl:3885]: This undoes a [previous commit from me](https://github.com/OpenLinkSoftware/glossaries/commit/cac9b5a2767189b21c2e144ef01b492d796ad95b) (applied by @openlink), which added the explicit expansion of the SPARQL acronym (which is not revealed by the current [old] text).

**Review by TallTed — COMMENTED** [review]
@openlink @kidehen --- Erroneous commits like this are part of why Pull Requests should never be merged immediately by the PR originator, but rather wait for commit until after review by at least one other human.  See my comments above for details.

File: GlossaryOfTerms.ttl (hunk 1 above)
Comment (openlink) [reply GlossaryOfTerms.ttl:42]: Fixed

**Pull request merged**